deployment/deployer.yaml

```diff
@@ -67,6 +67,16 @@ items:
       - watch
       - delete
       - update
+  - apiVersion: v1
```

Comment:
edit permission on nodes is an extremely high permission... how is that needed to scale up/down particular pods?